pkg/controller/catalogsourceconfig/configuring.go

```diff
@@ -5,14 +5,16 @@ import (
 	"fmt"
 	"strings"
 
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
 	olm "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
 	"github.com/operator-framework/operator-marketplace/pkg/apis/marketplace/v1alpha1"
 	"github.com/operator-framework/operator-marketplace/pkg/datastore"
 	"github.com/operator-framework/operator-marketplace/pkg/phase"
-	"github.com/operator-framework/operator-sdk/pkg/sdk"
-	"github.com/sirupsen/logrus"
+	log "github.com/sirupsen/logrus"
```

Comment:
The intention was to match the style that was defined by the generated controller. Otherwise, we are using two styles of log reference in each controller's package. If the former is preferable to the latter, I can revert it and just adjust the _controller.go files to match the old style.

Reply:
Sure but it shouldn't be in this PR. It makes the PR harder to review. If it should be in this PR, then it needs to be a separate commit.

Reply:
Done